numalogic/registry/artifact.py

```diff
@@ -35,6 +35,7 @@ def save(
         dkeys: Sequence[str],
         primary_artifact: Artifact,
         secondary_artifact: Artifact = None,
+        models_to_retain: int = 5,
```

Comment:
let's keep this as an argument to MlflowRegistrar constructor

Reply:
Got it.